extensions/data-plane/data-plane-azure-storage/README.md

```diff
@@ -86,3 +86,17 @@ An example destination address:
 The `folderName` and the `blobName` are optional properties in destination address.
 
 
+
+### AzureStorage Transfer Configuration
+
+The existing implementation takes under consideration transfer of files up to 200GB and that can be accomplished within the space of one hour. If your usage surpasses these limits or any other kind of additional tuning is needed, please [check this documentation](../../common/azure/azure-blob-core/README.md) containing the transfer configurations.
+
+#### File larger than 200GB
+
+To ease transfer of a large file, it is divided in several blocks. The default block size is 4MB which limits the maximum transfer file size to ~200GB, so, if there is a need to upload
```

Comment:
please explain why with a block size of 4MB the max size is 200GB and how changing the block size it can increase the max size.

Reply:
You're right, makes sense, added in [here](https://github.com/eclipse-edc/Technology-Azure/pull/298/commits/559b3dfd11a416d93c004ac0af10650f69fa1486).